pkg/golinters/forbidigo.go

```diff
@@ -47,7 +47,7 @@ func NewForbidigo(settings *config.ForbidigoSettings) *goanalysis.Linter {
 		nil,
 	).WithIssuesReporter(func(*linter.Context) []goanalysis.Issue {
 		return resIssues
-	}).WithLoadMode(goanalysis.LoadModeSyntax)
+	}).WithLoadMode(goanalysis.LoadModeTypesInfo)
```

Comment:
This commit can be removed from this PR. Let's merge https://github.com/golangci/golangci-lint/pull/3612 instead for a feature release of forbidigo because that PR enables new behavior.

Just updating forbidigo without additional code changes doesn't.

Reply:
This PR will be merged with my change as I explain here https://github.com/golangci/golangci-lint/pull/3612#discussion_r1118400388